tests/test_sdk/test_queryweaver.py

```
@@ -0,0 +1,482 @@
+            
+            # Validate results - should be 2 customers from New York
+            assert result.results is not None
+            assert isinstance(result.results, list)
+            assert len(result.results) == 2, f"Expected 2 customers from New York, got {len(result.results)}"
+            
+            # Verify the correct customer names are returned (Alice Smith and Carol White)
+            customer_names = [r.get("name") for r in result.results]
+            assert "Alice Smith" in customer_names, f"Expected 'Alice Smith' in results, got {customer_names}"
+            assert "Carol White" in customer_names, f"Expected 'Carol White' in results, got {customer_names}"
+            # Bob Jones should NOT be in results (he's from Los Angeles)
+            assert "Bob Jones" not in customer_names, f"'Bob Jones' should not be in NYC results"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)